src/server/worker/master.go

```diff
@@ -713,6 +713,9 @@ func (a *APIServer) waitJob(pachClient *client.APIClient, jobInfo *pps.JobInfo,
 			}
 		}
 		if jobInfo.EnableStats {
+			if jobInfo.StatsCommit == nil {
+				panic("on onoetenoentosuhaonteuhoaidu")
```

Comment:
We should do something other than panic here. Probably just log an error and move on without trying to finish the commit.

Reply:
haha whoops yeah